source/guides/guides/network-requests.md

```diff
@@ -404,3 +404,5 @@ You can find more examples in our {% url "XHR Assertions" https://github.com/cyp
 - {% url "Network requests in Kitchen Sink example" https://github.com/cypress-io/cypress-example-kitchensink/blob/master/cypress/integration/examples/network_requests.spec.js %}
 - {% url "See how to make a request with `cy.request()`" request %}
 - {% url "Real World App (RWA)" https://github.com/cypress-io/cypress-realworld-app %} test suites to see Cypress network handling in action.
+- Read {% url "Difference between cy.route and cy.route2" https://glebbahmutov.com/blog/cy-route-vs-route2/ %} blog post
```

Comment:
We're going to be renaming route2 to cy.http (hopefully in 6.0), just mentioning that we may want to consider updating the blog post for that.

Reply:
My blog post titles are like math constants - immutable